specifications/secure-launch-specification.rst

```diff
@@ -504,8 +518,21 @@ The list of valid flags for D-RTM Policy entries.
 .. code-block:: c
     :linenos: 1
 
-    #define SLR_POLICY_FLAG_MEASURED	0x1
-    #define SLR_POLICY_IMPLICIT_SIZE	0x2
+    #define SLR_POLICY_FLAG_MEASURED    0x1
+    #define SLR_POLICY_IMPLICIT_SIZE    0x2
+
+`SLR_POLICY_FLAG_MEASURED` **MAY** be used by DCE and/or DLME to mark which
+entries were measured, in case not all of them are measured at the same time.
+For example, limited in size DCE can use TPM commands for hashing instead of
+calculating the hashes to save space. DLME usually doesn't have strict size
+constraints, so it may include functions that are much faster than sending the
+data to be hashed by TPM. In such cases, `SLR_POLICY_FLAG_MEASURED` is set by
+DCE for entries it measures, and DLME skips those. Note that all entries still
+**MUST** be measured in order.
+
```

Comment:
This is specifically to handle the situation caused by the Linux TPM maintainers demanding that the mainline code be used if we wanted to do measurements from the compress kernel. This was not possible since the mainline driver makes use of functionality not available and thus creates a situation where we need to measure structures before they are used but don't have a means to access the TPM at the time.

So the general case is to handle situations where a measurement is needed but TPM access is not available yet. **BUT** the delay of sending the measurement cannot cross measured entity boundaries. So one component cannot measure another and then put  the event in the log to be sent to the TPM by that next component.

This should also be made a note block and place above the definitions.

Reply:
I suspected that to be the reason, included it as well. My original description was made mostly with SKL in mind.